threedi_modelchecker/checks/other.py

```diff
@@ -301,7 +288,7 @@ def get_invalid(self, session):
 class ConnectionNodesLength(BaseCheck):
     """Check that the distance between `start_node` and `end_node` is at least
     `min_distance`. The coords will be transformed into (the first entry) of
-    GlobalSettings.epsg_code.
+    ModelSettingss.epsg_code.
```

Comment:
```suggestion
    ModelSettings.epsg_code.
```
Fijn dat ik op deze manier toch iets kan bijdragen.